### PR TITLE
More test cases (POST request, connection with two streams)

### PR DESCRIPTION
**Pull request by jitsni**
Adding two new testcases:
1. POST request with payload
2. A HTTP2 connection has two streams

### CODE REVIEW

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newTargetInputRef1 ${nuklei:newReferenceId()} # external scope
+property newTargetInputRef2 ${nuklei:newReferenceId()} # external scope
```
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt]: Easier to have both streams use the same `newTargetInputRef` (but different `newTargetInputId`s) when integrating implementation testing with the control script that sets up the route.

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+write ${newSourceInputId}
+write [0x00 0x18]
+write [0x00 0x00 0x0f]                                                  # length = 15
+      [0x01]                                                            # HEADERS frame
+      [0x05]                                                            # END_STREAM | END_HEADERS
+      [0x00 0x00 0x00 0x01]                                             # stream_id = 1
+      [0x83]                                                            # :method: POST
+      [0x86]                                                            # :scheme: http
+      [0x84]                                                            # :path: /
+      [0x41] [0x8a 0xa0 0xe4 0x1d 0x13 0x9d 0x09 0xb8 0xf0 0x1e 0x07]   # :authority: localhost:8080
```
Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt]: I think it is huffman encoded

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+# data frame - HTTP2 DATA stream_id=1
+write [0x00 0x00 0x00 0x02]
+write ${newSourceInputId}
+write [0x00 0x10]
+write [0x00 0x00 0x07]                                                  # length = 15
+      [0x00]                                                            # HTTP2 DATA frame
+      [0x01]                                                            # END_STREAM
+      [0x00 0x00 0x00 0x01]                                             # stream_id = 1
+      "foo1bar"
```
Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt]: done

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt (hunk 1 above)
Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt]: done

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+write flush
+
+# data frame (http2 client connection preface)
+read [0x00 0x00 0x00 0x02]
+read ${sourceInputId}
+read [0x00 0x18]
+read "PRI * HTTP/2.0\r\n"
+      "\r\n"
+      "SM\r\n"
+      "\r\n"
```
Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt]: done

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt (hunk 2 above)
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt]: Can we make it in the clear instead for readability please?
The Huffman encoding tests can be done once separately, and the other tests can all remain in the clear.

Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt]: done

File: src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt [hunk 5]
```diff
@@ -104,15 +104,15 @@ write flush
...
+read [0x00 0x1c]
+read [0x00 0x00 0x13]                                                   # length = 19
      [0x01]                                                             # HEADERS frame
      [0x05]                                                             # END_STREAM | END_HEADERS
      [0x00 0x00 0x00 0x01]                                              # stream_id = 1
      [0x83]                                                             # :method: POST
      [0x86]                                                             # :scheme: http
      [0x84]                                                             # :path: /
-     [0x41] [0x8a 0xa0 0xe4 0x1d 0x13 0x9d 0x09 0xb8 0xf0 0x1e 0x07]    # :authority: localhost:8080
+     [0x01] [0x0e] "localhost:8080"                                     # :authority: localhost:8080
```
Comment (jfallows) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt]: Please fix the indentation of the per-line comments, previously forced wide by the huffman byte sequence.

Comment (jitsni) [inline src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt]: done

**Pull request merged**